tests/yam/agama/boot_agama.pm

```diff
@@ -30,6 +31,11 @@ sub run {
         $self->bootloader_s390::run();
         return;
     }
+    elsif (is_s390x() && check_var("BACKEND", "svirt")) {
```

Comment:
Please, merge with the code above, putting is_s390x() to wrap the other two conditions, check if there is something similar for is_backend_svirt, maybe it can be created, and put the `return` instruct shared to not repeat it.

Reply:
Updated, thanks.